File: java/src/test/java/com/adaptionsoft/games/trivia/GameTest.java
package com.adaptionsoft.games.trivia;

import org.junit.jupiter.api.Test;

public class GameTest {

    @Test
    public void triviaGameForDataSetOne() {
        ApprovalGame.create(5L).runTestScenario();
    }

    @Test
    public void triviaGameForDataSetTwo() {
        ApprovalGame.create(7L).runTestScenario();
    }

    @Test
    public void triviaGameForDataSetThree() {
        ApprovalGame.create(13L).runTestScenario();
    }

    @Test
    public void triviaGameForDataSetFour() {
        ApprovalGame.create(17L).runTestScenario();
    }

    @Test
    public void triviaGameForDatSetFive() {
        ApprovalGame.create(93L).runTestScenario();
    }


}

File: kotlin/src/test/kotlin/com/adaptionsoft/games/trivia/GameTest.kt
package com.adaptionsoft.games.trivia

import com.adaptionsoft.games.trivia.ApprovalGame.Companion.create
import org.junit.jupiter.api.*

internal class GameTest {
    @Test
    fun triviaGameForDataSetOne() {
        create(5L).runTestScenario()
    }

    @Test
    fun triviaGameForDataSetTwo() {
        create(7L).runTestScenario()
    }

    @Test
    fun triviaGameForDataSetThree() {
        create(13L).runTestScenario()
    }

    @Test
    fun triviaGameForDataSetFour() {
        create(17L).runTestScenario()
    }

    @Test
    fun triviaGameForDatSetFive() {
        create(93L).runTestScenario()
    }
}